dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.cdi4.1-expressionLanguage6.0.feature

```diff
@@ -0,0 +1,10 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.cdi4.1-expressionLanguage6.0
+Manifest-Version: 1.0
+IBM-API-Package: jakarta.enterprise.inject.spi.el;  type="spec"
+IBM-Provision-Capability: \
+  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=io.openliberty.cdi-4.1.feature))", \
```

Comment:
I suspect this will not work because the feature name should not have `.feature` on the end of it. I didn't spot this during review but will fix it when I add a test.